xchange-binance/src/test/java/org/knowm/xchange/binance/AbstractResilienceTest.java

```diff
@@ -29,6 +29,10 @@ protected BinanceExchange createExchangeWithRateLimiterEnabled() {
     return createExchange(false, true);
   }
 
+  protected BinanceExchange createExchange() {
```

Comment:
this is the same as createExchangeWithRetryDisabled and therefore redundant

Reply:
removed